### PR TITLE
Cleanup readme

### LINKED ISSUE #613: Clean up the README to make it clear
That was a mistake of me to keep the list of [MeiliSearch raw supported features under the "Technical features" header](https://github.com/meilisearch/MeiliSearch#technical-features). I would like that no more links points to the internal code.

This is more disturbing than helpful for anyway.

Anyway, would it be possible to clear and shorten the README, removing one of the two gifs, the performances section could be moved to the documentation for example.

I didn't thought that people comming to the repository would be people who don't want to know about the internal tech but just want to see the supported features for their needs.

What do you think about that?

### PR DESCRIPTION
**Pull request by tpayet**
Closes #613

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,54 +26,52 @@
...
+#### APT (Debian & Ubuntu)
 
 ```bash
 echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list
 apt update && apt install meilisearch-http
 meilisearch
 ```
 
-#### Download the binary
+#### Download the binary (Linux & Mac OS)
```
Comment (Kerollmops) [inline README.md:74]: We also support Windows now :)

File: README.md [hunk 2]
```diff
@@ -164,33 +161,31 @@ We also deliver an **out-of-the-box web interface** in which you can test MeiliS
 
 You can access the web interface in your web browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700). All you need to do is open your web browser and enter MeiliSearch’s address to visit it. This will lead you to a web page with a search bar that will allow you to search in the selected index.
 
-<p align="center">
-  <img src="assets/movies-web-demo.gif" alt="Web interface gif" />
-</p>
+| [See the gif above](#demo)
```
Comment (Kerollmops) [inline README.md:164]: People that came here probably already saw the gif, so I would prefer removing this sentence.

```suggestion
```

File: README.md [hunk 3]
```diff
@@ -164,33 +161,31 @@ We also deliver an **out-of-the-box web interface** in which you can test MeiliS
...
-### Analytic Events
+## Telemetry
 
-Every hour, events are being sent to our Amplitude instance so we can know how many people are using MeiliSearch.<br/>
+MeiliSearch collects anonymous data regarding general usage.
+This helps us better understand developers usage of MeiliSearch features.<br/>
 To see what information we're retrieving, please see the complete list [on the dedicated issue](https://github.com/meilisearch/MeiliSearch/issues/720).<br/>
 We also use Sentry to make us crash and error reports. If you want to know more about what Sentry collects, please visit their [privacy policy website](https://sentry.io/privacy/).<br/>
-If this doesn't suit you, you can disable these analytics by using the `MEILI_NO_ANALYTICS` env variable.
+This program is optionnal, you can disable these analytics by using the `MEILI_NO_ANALYTICS` env variable.
```
Comment (Kerollmops) [inline README.md:180]: What do you mean by "this program", this env variable disable all the analytics, sentry and our own analytics.

File: README.md [hunk 4]
```diff
@@ -164,33 +161,31 @@ We also deliver an **out-of-the-box web interface** in which you can test MeiliS
...
 
-## Contact
+## 💌 Contact
 
 Feel free to contact us about any questions you may have:
-* At [bonjour@meilisearch.com](mailto:bonjour@meilisearch.com): English or French is welcome! 🇬🇧 🇫🇷
+* At [bonjour@meilisearch.com](mailto:bonjour@meilisearch.com)
```
Comment (Kerollmops) [inline README.md:185]:
```suggestion
* At [bonjour@meilisearch.com](mailto:bonjour@meilisearch.com).
```

File: README.md [hunk 5]
```diff
@@ -164,33 +161,31 @@ We also deliver an **out-of-the-box web interface** in which you can test MeiliS
...
 * Via the chat box available on every page of [our documentation](https://docs.meilisearch.com/) and on [our landing page](https://www.meilisearch.com/).
-* 🆕 Join our [GitHub Discussions forum](https://github.com/meilisearch/MeiliSearch/discussions) (BETA hype!)
+* 🆕 Join our [GitHub Discussions forum](https://github.com/meilisearch/MeiliSearch/discussions)
```
Comment (Kerollmops) [inline README.md:187]:
```suggestion
* 🆕 Join our [GitHub Discussions forum](https://github.com/meilisearch/MeiliSearch/discussions).
```

File: README.md (hunk 1 above)
Comment (tpayet) [reply README.md:74]: Not from the download-latest script though

File: README.md (hunk 3 above)
Comment (tpayet) [reply README.md:179]: The telemetry program.

File: README.md [hunk 6]
```diff
@@ -27,54 +26,52 @@
 **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box.
 For more information about features go to [our documentation](https://docs.meilisearch.com/).
 
-<p align="center">
-  <a href="https://crates.meilisearch.com"><img src="assets/crates-io-demo.gif" alt="crates.io demo gif" /></a>
+<p align="center" id="demo">
```
Comment (Kerollmops) [inline README.md:29]: Why have you added an `id`?

Comment (tpayet) [reply README.md:29]: It was for the link that we suppressed, I just remove id

**Pull request merged**